pkgs/games/bugdom/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, cmake, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "bugdom";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "jorio";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:1371inw11rzfrxmc3v4gv5axp56bxjbcr0mhqm4x839401bfq5mf";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    SDL2
+    cmake
+    makeWrapper
+  ];
+
+  configurePhase = ''
+    cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
+  '';
+
+  buildPhase = ''
+    cmake --build build
+  '';
+
+  installPhase = ''
+    mv build $out
+    makeWrapper $out/Bugdom $out/bin/Bugdom --run "cd $out"
+  '';
```

Comment:
All of these phases should run `runHook pre/postBuild/Configure/Install` in order to make overriding easier.

Alternatively, you could also use `postConfigure` instead of `configurePhase`, and the same for build and install.